connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/ReplicationPolicy.java

```diff
@@ -52,9 +52,39 @@ default String originalTopic(String topic) {
         }
     }
 
+    /** Returns heartbeats topic name.*/
+    default String heartbeatsTopic() {
+        return "heartbeats";
+    }
+
+    /** Returns the offset-syncs topic for given cluster alias. */
+    default String offsetSyncTopic(String clusterAlias) {
```

Comment:
Should we called this method `offsetSyncsTopic` so it matches the actual topic name?